scripts/generate-database.js

```diff
@@ -333,6 +333,13 @@ const extractExamples = async (
           ...eventsBasedExtensions.map(({ fullName }) => fullName),
         ];
         tags.forEach((tag) => allExampleTags.add(tag));
+        const difficultyLevel = tags.includes('simple')
+          ? 'simple'
+          : tags.includes('advanced')
+          ? 'advanced'
+          : tags.includes('expert')
+          ? 'expert'
+          : undefined;
```

Comment:
I didn't add a "medium" level because I don't know know any example that would fell in the level.
Examples are either "simple" or "advanced". I put a few examples that are so complicated that I think users will prefer to find their own way rather than trying to understand the example as "expert".

Reply:
I like that, expert is for people who want to understand code rather than develop a game